CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Fixed
 * Fixed an issue that would result in UWP apps being rejected from the Microsoft Store due to an unsupported API (`__C_specific_handler`) being used. (Issue [#2235](https://github.com/realm/realm-dotnet/issues/2235))
 * Fixed a bug where applying multiple `OrderBy` clauses on a query would result in the clauses being appended to each other as if they were `.ThenBy` rather than the last clause replacing the preceding ones. (PR [#2255](https://github.com/realm/realm-dotnet/issues/2255))
+* Fixed proper selection of Additive mode (Explicit or Discovered) to pass to the schema builder. This selection is based on whether or not the user has provided a subset of classes to the Realm Schema. More info about this subject can be found [here](https://docs.mongodb.com/realm/dotnet/objects/#provide-a-subset-of-classes-to-your-realm-schema). (PR [#2259](https://github.com/realm/realm-dotnet/pull/2259))
```

Comment:
2 issues with this:

1. I'm not sure this should be considered an issue. After all, it was consciously decided to have a quick change to the breaking changes introduced by the splitting of the schema mode `Additive`. If we agree on this, maybe internal could be a better section. Although, even *internal* means like a change when this must be addressed given the new change in core. So, please advice if this is necessary at all and where it should go.
2. I'm not that happy to have the PR# here just yet. This is because another PR may be needed based on the outcome of the conversation with James.